macros/dmt_unit_test.sql

```diff
@@ -31,7 +31,9 @@
         {%- set exclude_columns = [] -%}
         {%- for col in all_columns -%}
             {%- set col = col|replace('"',"") -%}
-            {%- if col not in compare_columns|upper -%}
+            {# -- in bigquery columns seem to come quoted with ` #}
+            {%- set col = col|replace('`',"") -%}
```

Comment:
Instead of adding this line I think you could just edit line 33 instead - worked for me

Reply:
resolving this to merge 👍